### DISCUSSION TITLE
Playcover unable to open from application folder

### DISCUSSION THREAD

**Discussion by yascore** [PlayCover/PlayCover#771] [Q&A] [answered]
HI !
I am trying to open playcover from the application folder but when i click it, the playcover icon show up crossed out and a window opens saying "impossible to open play cover on this Mac". I don't get it because Playcover is supposed to work on Mac OS 12 and more and i have Mac OS 12.6.3 the latest version. Do you have any idea what's the problem or can it be because my macbook air is from 2018?
Thanks for your answers if you can help me !

**Comment by ohaiibuzzle** [comment, marked as answer]
Yes. PlayCover does not work with your Mac. It requires an Apple chip to work (so Macs released after November, 2020)
